odk2-src/getting-started-2-architect.rst

```diff
@@ -1,214 +1,39 @@
-ODK 2.0 Getting Started Guide
-===============================
+ODK 2 Getting Started Deployment Architect Guide
+==========================================================
 
-.. _using-odk-2:
+.. _architect-odk-2:
 
-The ODK 2.0 tools are intended to address limitations of the existing tool set. The 2.0 Tool Suite consists of:
-
-- :doc:`services-intro` - an application that handles database access, file access, and data synchronization services between all of the ODK 2.0 applications. It also allows you to synchronize data collected by the ODK 2.0 tools using the 2.0 protocol with an ODK Aggregate instance.
-- :doc:`survey-intro` - a data collection application based upon HTML, CSS, and JavaScript.
-- :doc:`tables-intro` - a data collection and visualization application running on your device.
-- :doc:`cloud-endpoints-intro` - a ready-to-deploy server and data repository with enhancements to support bi-directional data synchronization across disconnected devices.
-- :doc:`app-designer-intro` - a design environment for creating, customizing, and previewing your forms.
-- :doc:`suitcase-intro` - a desktop tool for synchronizing data from an ODK 2.0 server so the data can be exported to CSV format.
-
-This page provides a brief end-to-end walk-through of the ODK 2.0 tools. It will cover the following topics:
+This guide is intended for Deployment Architects (see the :ref:`odk-2-perspectives` section for details).
```

Comment:
I think you should put the Deployment Architect description here and link if they want to find out more. Otherwise people will be constantly cross linking and have a hard time keeping on the page they were on IMHO

Reply:
Yes I noticed this after the PR. I've got some stuff that will hopefully clarify that I'll add.